config/webpack/plugin/TerraDevSite.js

```diff
@@ -69,6 +70,7 @@ class TerraDevSite {
     new GeneratePlugin({
       sites: this.sites,
       basename,
+      inTest: this.inTest,
```

Comment:
What about `USE_TEST_NAV_CONFIG`?

Reply:
i like inTest better than that ;P

Reply:
Also, I'd like to keep the `test mode` env generic because in the future we may do more than just flex the nav config when we're in test mode.

Reply:
hmmm.. lemme think on that
If there might be other uses, it might be helpfully to pull fallback to a testConfig instead of a testNavConfig.

Reply:
Sorry...been sitting on this for a while. Thoughts on: `TERRA_ENV`, `useTestConfig`, idk I don't love the `inTest` env but maybe it's just me

overall I think the changes look good 👍

Reply:
I'm going to switch the env to `testMode`